packages/@textlint/markdown-to-ast/test/fixtures/auto-link-url.text/output.json

```diff
@@ -204,22 +204,22 @@
             "children": [
                 {
                     "type": "Str",
-                    "value": "So should this: ",
+                    "value": "So should this: mailto:",
```

Comment:
https://github.com/remarkjs/remark/issues/475
mailto:example@examle.com is changed